### ISSUE TITLE
Optional red bars for signed messages

### ISSUE THREAD

**Issue by Madis0** [Aizistral-Studios/No-Chat-Reports#373]
### Idea

Essentially the opposite of what Minecraft 1.19.2 did - what if NCR added an option to show signed messages with a red bar beforehand?

### Reasoning

- Another reminder that the server allows reporting
- A way to see which players don't have NCR (if it's optional on the server)
- Debugging purposes

### Other Information

Disabled by default

**Comment by Aizistral** [comment]
Certainly worth trying to implement, especially since we already have a framework for adding custom chat indicators.

**Comment by IlyaTheSawmill** [comment]
There can be a yellow bar for player messages and red bar for sigened messages.
This can remind players that player messages are still existing.

**Comment by Madis0** [comment]
> There can be a yellow bar for player messages and red bar for sigened messages. This can remind players that player messages are still existing.

What do you mean by "player messages"? And why yellow?

**Comment by Aizistral** [comment]
> What do you mean by "player messages"? And why yellow?

I suppose they mean the technical separation between player messages and system messages. I don't think unsigned player messages should require any special indicators though, they don't carry any significant information that is not present in system messages.

**Comment by IlyaTheSawmill** [comment]
> I suppose they mean the technical separation between player messages and system messages. I don't think unsigned player messages should require any special indicators though, they don't carry any significant information that is not present in system messages.

Yeah, I want to separate unsigned and system messages by seeing the color of the bar, but please use _he_ instead of _they_ to refer to me.
